fe/fe-core/src/main/java/com/starrocks/server/HiveTableFactory.java

```diff
@@ -60,17 +74,8 @@ public Table createTable(LocalMetastore metastore, Database database, CreateTabl
         HiveTable.Builder tableBuilder = HiveTable.builder()
                 .setId(tableId)
                 .setTableName(tableName)
-                .setCatalogName(oHiveTable.getCatalogName())
-                .setResourceName(oHiveTable.getResourceName())
-                .setHiveDbName(oHiveTable.getDbName())
-                .setHiveTableName(oHiveTable.getTableName())
-                .setPartitionColumnNames(oHiveTable.getPartitionColumnNames())
-                .setDataColumnNames(oHiveTable.getDataColumnNames())
-                .setFullSchema(columns)
-                .setTableLocation(oHiveTable.getTableLocation())
-                .setStorageFormat(oHiveTable.getStorageFormat())
-                .setCreateTime(oHiveTable.getCreateTime())
-                .setProperties(oHiveTable.getProperties());
+                .setFullSchema(columns);
+        copyFromOldTable(tableBuilder, oHiveTable, properties);
```

Comment:
The original meaning of oHiveTable is `other table`, not `old table`. It may be more reasonable to name it `tableFromCatalog` ?

Reply:
Fixed.  I've chhanged to `catalogTable`.